docs/sphinx/source/index.rst

```diff
@@ -10,14 +10,14 @@ energy systems and accomplishing related tasks.
 The core mission of pvlib python is to provide open,
 reliable, interoperable, and benchmark implementations of PV system models.
 
-The source code for pvlib python is hosted on `github
+The source code for pvlib python is hosted on `GitHub
 <https://github.com/pvlib/pvlib-python>`_.
 Please see the :ref:`installation` page for installation help.
 
 For examples of how to use pvlib python, please see
-:ref:`package_overview` and our `Jupyter Notebook tutorials
-<http://nbviewer.org/github/pvlib/pvlib-python/tree/main/docs/
-tutorials/>`_. The documentation assumes general familiarity with
+:ref:`package_overview` and the `Example Gallery
+<https://pvlib-python.readthedocs.io/en/stable/gallery/index.html/>`_.
```

Comment:
```suggestion
:ref:`package_overview` and the :ref:`example_gallery`.
```

Reply:
Was about to say the same thou I didn't know how to suggest changes (now I do 🤠 )

In case it helps somebody, we are seeing an explicit target being suggested. They are identified because they are like a directive, but with simple colon and start with a underscore.

https://github.com/pvlib/pvlib-python/blob/c2ac0220cfa5717e6f257f930e793a3d76ad28c9/docs/examples/README.rst?plain=1#L1

[RTD Documentation](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets) regarding links.